docs/dsp-tools-excel.md

```diff
@@ -58,15 +58,16 @@ The Excel sheet must have the following structure:
 
 The expected columns are:
 
-- `name` : The name of the property
-- `super` : The base property of the property
-- `object` : If the property is derived from `hasValue`, the type of the property must be further specified by the 
+- `name`: The name of the property
+- `super`: The base property/ies of the property, separated by commas
+- `object`: If the property is derived from `hasValue`, the type of the property must be further specified by the 
 object it takes, e.g. `TextValue`, `ListValue`, or `IntValue`. If the property is derived from `hasLinkTo`, 
 the `object` specifies the resource class that this property refers to.
-- `en`, `de`, `fr`, `it` : The labels of the property in different languages, at least one language has to be provided
+- `en`, `de`, `fr`, `it`: The labels of the property in different languages, at least one language has to be provided
 - `comment_en`, `comment_de`, `comment_fr`, `comment_it`: optional comments in the respective language 
-- `gui_element` : The GUI element for the property
-- `hlist` : In case of list values: the name of the list
+- `gui_element`: The GUI element for the property
+- `gui_attributes` (optional): The gui_attributes in the form "attr: value, attr: value". In case of ListValues, the 
+  name of the list can be given as "hlist: listname" (according to the pattern), or simply as "listname".
```

Comment:
For backwards compatibility, I renamed the last column from 'hlist' to 'gui_attributes', and allowed the simple 'listname' instead of 'hlist: listname'. Like this, this PR doesn't break old excel files.
It is necessary to support gui_attributes, because they are required in some cases.